doc/working-with-spyder.rst

```diff
@@ -22,3 +22,39 @@ Explore how to take advantage of Spyder’s functionality beyond just the four c
 * Quickly navigate within and between files with the Outline pane
 * Search for text or regular expressions across your entire project with the Find pane
 * Discover and explore structured documentation in the Online Help pane
+
+
+
+===========================
+Improving your code quality
+===========================
+
+Learn how to improve the quality of your programs using code analysis.
+
+.. youtube:: JnljnAjdO_w
+            :height: 360
+            :width: 640
+            :align: left
+
+* Open and use Code Analysis to evaluate the quality and style of Python files
+* Run analysis on a file in the Editor, on your computer or at a path
+* Determine what an error, warning or message means and how to fix it
+* Turn off messages on a line, in a file or globally
+
+
+
+====================
+Optimizing your code
+====================
+
+Learn how to optimize your code using the profiler.
+
+.. youtube:: 4hcXa8aGu_c
+            :height: 360
+            :width: 640
+            :align: left
+
+* Open and utilize the Profiler to determine the bottlenecks in your programs
+* Run profiling on a file in the Editor, on your computer or at a path
```

Comment:
```suggestion
* Use the Profiler to determine bottlenecks in your programs
* Run profiling on a file in the Editor or in your computer
```

Reply:
Same as before, I'll wait for @CAM-Gerlach to see if "on" is right.

Reply:
```suggestion
* Use the Profiler to find bottlenecks in your programs
* Run profiling on a file in the Editor or elsewhere on your machine
```

Yep, same as before @juanis2112 is correct, and I agree with simplifying it. I changed "determine" to "find" since it was a more appropriate word with the revised phrasing (determine is a more complicated word and needs "the"), and made the wording of the second phrase less duplicative of the previous video (my fault originally).